sql/src/test/java/io/crate/integrationtests/SysNodeCheckerIntegrationTest.java

```diff
@@ -36,10 +45,28 @@
 @UseJdbc
 public class SysNodeCheckerIntegrationTest extends SQLTransportIntegrationTest {
 
+    private static List<String> dirs = new ArrayList<>();
+
+    @Override
+    // TODO: This is for testing the ClusterNameInPathDataNodesSysCheck and
```

Comment:
Maybe rename all TODOs in this PR to ES6-TODO or something like that. We've already a lot of other TODOS and so it will at least be easier to grep for these specifically